src/Sulu/Bundle/AutomationBundle/Resources/config/doctrine/Task.orm.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+    <entity name="Sulu\Bundle\AutomationBundle\Entity\Task" table="au_task">
+        <id name="id" type="integer" column="id">
+            <generator strategy="AUTO"/>
+        </id>
+
+        <field name="entityClass" type="string" column="entityClass"/>
+        <field name="entityId" type="string" column="entityId"/>
+
+        <field name="taskName" type="string" column="taskName"/>
+        <field name="schedule" type="datetime" column="schedule"/>
```

Comment:
we have currently no recuring events. but then we would have to add some additional fields here (like cron-string or other stuff).